src/api/ITooltip.js

```diff
@@ -109,7 +109,12 @@
         if (opts.value instanceof Date) {
             opts.value = opts.value || "";
         } else {
-            opts.value = this._valueFormatter(opts.value) || "";
+            if(typeof opts.value !== undefined){
+                opts.value = this.fields()[1].transform(opts.value);
```

Comment:
I added this condition only for Tooltip. Tooltip will be called only when there is a data so without data Tooltip will not be called. I have added typeof opts.value condition incase if there is a chance for empty value.